<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>

  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>

   <property name="styleSheet">
    <string notr="true">
QWidget#bgwidget[theme="light"] {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:1,
                                      stop:0 #DCC7AA,
                                      stop:1 #FFFFFF);
}
QWidget#bgwidget[theme="dark"] {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:1,
                                      stop:0 #8B5E3C,
                                      stop:1 #F5F1E6);
}

QWidget#bgwidget QLabel {
    font: 11pt "Segoe UI";
}
QWidget#bgwidget[theme="light"] QLabel {
    color: #3A2A20;
}
QWidget#bgwidget[theme="dark"] QLabel {
    color: #F5F1E6;
}

QWidget#bgwidget QLabel#label {
    font: 700 32pt "Segoe UI";
}

QWidget#bgwidget QLabel#error {
    font: 10pt "Segoe UI";
    color: #FF4D4D;
}

QWidget#bgwidget[theme="light"] QLineEdit {
    border-radius: 10px;
    padding: 6px 10px;
    font: 11pt "Segoe UI";
    border: 1px solid #C97B5A;
    background-color: #FFFFFF;
    color: #3A2A20;
}
QWidget#bgwidget[theme="dark"] QLineEdit {
    border-radius: 10px;
    padding: 6px 10px;
    font: 11pt "Segoe UI";
    border: 1px solid #F5F1E6;
    background-color: #8B5E3C;
    color: #F5F1E6;
}
QWidget#bgwidget QLineEdit:focus {
    border: 1px solid #C97B5A;
}

QWidget#bgwidget[theme="light"] QPushButton#login {
    border-radius: 20px;
    font: 600 12pt "Segoe UI";
    padding: 8px 12px;
    border: none;
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #C97B5A,
                                      stop:1 #B06345);
    color: #FFFFFF;
}
QWidget#bgwidget[theme="light"] QPushButton#login:hover {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #D18764,
                                      stop:1 #BD6D4F);
}
QWidget#bgwidget[theme="light"] QPushButton#login:pressed {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #A3573C,
                                      stop:1 #8B4731);
}

QWidget#bgwidget[theme="dark"] QPushButton#login {
    border-radius: 20px;
    font: 600 12pt "Segoe UI";
    padding: 8px 12px;
    border: none;
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #8B5E3C,
                                      stop:1 #F5F1E6);
    color: #8B5E3C;
}
QWidget#bgwidget[theme="dark"] QPushButton#login:hover {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #9C6B45,
                                      stop:1 #F5F1E6);
}
QWidget#bgwidget[theme="dark"] QPushButton#login:pressed {
    background-color: qlineargradient(spread:pad,
                                      x1:0, y1:0,
                                      x2:1, y2:0,
                                      stop:0 #754B30,
                                      stop:1 #E4D7C1);
}

QWidget#bgwidget[theme="light"] QPushButton#backToWelcome {
    background-color: transparent;
    border: none;
    border-radius: 0;
    font: 10pt "Segoe UI";
    padding: 0px;
    text-align: center;
    color: #C97B5A;
    text-decoration: underline;
}
QWidget#bgwidget[theme="dark"] QPushButton#backToWelcome {
    background-color: transparent;
    border: none;
    border-radius: 0;
    font: 10pt "Segoe UI";
    padding: 0px;
    text-align: center;
    color: #F5F1E6;
    text-decoration: underline;
}
QWidget#bgwidget QPushButton#backToWelcome:hover {
    opacity: 0.8;
}

QWidget#bgwidget QPushButton#themeToggle {
    font: 12pt "Segoe UI";
    padding: 6px 12px;
    border-radius: 14px;
    border: 1px solid #8B5E3C;
}

QWidget#bgwidget[theme="light"] QPushButton#themeToggle {
    color: #8B5E3C;
    background-color: rgba(255, 255, 255, 150);
}
QWidget#bgwidget[theme="light"] QPushButton#themeToggle:hover {
    background-color: rgba(255, 255, 255, 200);
}

QWidget#bgwidget[theme="dark"] QPushButton#themeToggle {
    color: #F5F1E6;
    border: 1px solid #F5F1E6;
    background-color: rgba(0, 0, 0, 40);
}
QWidget#bgwidget[theme="dark"] QPushButton#themeToggle:hover {
    background-color: rgba(0, 0, 0, 80);
}
    </string>
   </property>

   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>500</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign in to your existing account</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <widget class="QLineEdit" name="emailfield">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Username or email</string>
    </property>
   </widget>

   <widget class="QLineEdit" name="passwordfield">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>

   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>

   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>456</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>

   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>

   <widget class="QPushButton" name="backToWelcome">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>560</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Back to Welcome</string>
    </property>
   </widget>

    <!-- BETTER THEME TOGGLE BUTTON -->
   <widget class="QPushButton" name="themeToggle">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle theme</string>
    </property>
   </widget>

   <widget class="QLabel" name="messageLabel">
 <property name="geometry">
  <rect>
   <x>430</x>
   <y>260</y>
   <width>391</width>
   <height>41</height>
  </rect>
 </property>
 <property name="text">
  <string></string>
 </property>
 <property name="alignment">
  <set>Qt::AlignCenter</set>
 </property>
 <property name="styleSheet">
  <string notr="true">font: 12pt "Segoe UI"; color: #C97B5A;</string>
 </property>
</widget>

  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
